apps/api/src/middlewares/auth.ts

```
@@ -0,0 +1,118 @@
+  .use(
+    elysiaJwt({
+      name: "jwt",
+      secret: jwks as unknown as any, // Remote JWKS resolver (jose)
+    }),
+  )
+
+  /* ──────────────────────────────────────────────────────────────── */
+  /* 3) Accept exactly one credential                                */
+  /* ──────────────────────────────────────────────────────────────── */
+  .onBeforeHandle({ as: "global" }, async (ctx) => {
+    const { headers, bearer: bearerToken, jwt, set, store } = ctx as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)